### DISCUSSION TITLE
School's Designated Email Address

### DISCUSSION THREAD

**Discussion by Manju-S-6271** [community/community#112154] [GitHub Education] [answered]
### Hi everyone,

Hello. I entered my Email address and school name in the registration form because I want to participate in the GitHub Education program, and I got the following message.

```
Please select a different email address.

We require applicants of **************** to use one of these school-issued email addresses to apply:
- ***********
```

The domain displayed here is not assigned by our school. This domain appears to be a privately owned domain belonging to a student at our school who is graduating this year. What should I do?

**Comment by jcilacad** [comment]
If the domain you are using is not from your school, you can check this discussion about it.

https://github.com/orgs/community/discussions/76874

**Reply by Manju-S-6271** [reply]
Thanks to answer, but My problem is happening because when someone else previously registered, he registered with a domain that he himself owned.
Therefore, that Discussion is not helpful to me.

**Comment by Manju-S-6271** [comment, marked as answer]
Contacting GitHub Support resolved the issue.
